int32 botID
string direction
int32 currentX
int32 currentY
---
bool success